core/src/main/java/ch/cyberduck/core/transfer/download/AbstractDownloadFilter.java

```diff
@@ -282,6 +282,8 @@ public void complete(final Path file, final Local local,
                 if(local.exists()) {
                     local.delete();
                 }
+                listener.message(MessageFormat.format(LocaleFactory.localizedString("Finalize {0}", "Status"),
```

Comment:
Can we make this more userfriendly, and include ProgressReporting capabilities to the target-file WriteChannel?